core/src/main/scala/scala/pickling/generator/Macros.scala

```diff
@@ -3,9 +3,17 @@ package generator
 
 private[pickling] object PicklingMacros {
   import scala.language.experimental.macros
-  def genPickler[T]: Pickler[T] with Generated = macro scala.pickling.generator.Compat.genPickler_impl[T]
-  def genUnpickler[T]: Unpickler[T] with Generated = macro scala.pickling.generator.Compat.genUnpickler_impl[T]
-  def genPicklerUnpickler[T]: AbstractPicklerUnpickler[T] with Generated = macro scala.pickling.generator.Compat.genPicklerUnpickler_impl[T]
+
+  @deprecated("Use `genPicklerUnpickler` instead", "0.11")
+  def genPickler[T]: AbstractPicklerUnpickler[T] with Generated =
+    macro scala.pickling.generator.Compat.genPicklerUnpickler_impl[T]
+
+  @deprecated("Use `genPicklerUnpickler` instead", "0.11")
+  def genUnpickler[T]: AbstractPicklerUnpickler[T] with Generated =
+    macro scala.pickling.generator.Compat.genPicklerUnpickler_impl[T]
+
+  def genPicklerUnpickler[T]: AbstractPicklerUnpickler[T] with Generated =
```

Comment:
I feel like now (0.11 release) is the time to give this a name besides "PicklerUnpickler" but it can wait for a new CL.

Ideas for a name besides "PicklerUnpickler".   Previous ideas:  `Format`, `Structure`, `Juice` (get it? pickle juice?)